src/mmrelay/plugin_loader.py

```
@@ -77,17 +130,47 @@ def get_community_plugin_dirs():
     return _get_plugin_dirs("community")
 
 
+def _run(cmd, timeout=120, **kwargs):
+    # Validate command to prevent shell injection
+    if not isinstance(cmd, list):
+        raise ValueError("Command must be a list, not a string or other type")
```

Ruff (0.13.1)

Prefer `TypeError` exception for invalid type

(TRY004)